src/runtime/contributions-flow.js

```diff
@@ -82,8 +82,20 @@ export class ContributionsFlow {
    */
   startPayFlow_(response) {
     const sku = response.getSku();
+    const isOneTime = response.getOneTime();
     if (sku) {
-      new PayStartFlow(this.deps_, sku, ProductType.UI_CONTRIBUTION).start();
+      if (isOneTime) {
+        const contributionRequest = {};
+        contributionRequest.skuId = sku;
+        contributionRequest.oneTime = isOneTime;
```

Comment:
Could simplify as:

const contributionRequest = {
  skuId: sku,
  oneTiem: isOnetime,
}

Reply:
Fixed, thank you!